test/integration/scheduler/framework_test.go

```diff
@@ -1596,8 +1596,9 @@ func TestPreemptWithPermitPlugin(t *testing.T) {
 	}
 
 	// Create two pods.
-	waitingPod, err := createPausePod(cs,
```

Comment:
If it is all right, `waitingPod` will never be bound to a node, so the graceful delete time will be zero by apiserver, I think we do not need to zero it explicitly.

Reply:
Why do you think it can’t be bound to a node? Setting grace period deletion time to 0 in client is to make sure the deletion happens immediately apiserver receives the deletion request.

Reply:
If apiserver can detect it as an unbound pod and set the grace period to 0, then it makes sense to me, and we may need to dig more on the flake.

However, in code's perspective, explicitly outweighs ambiguity.